chainio/txmgr/metrics.go

```diff
@@ -0,0 +1,108 @@
+package txmgr
+
+import (
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const namespace = "txmgr"
+
+type Metrics struct {
```

Comment:
How is this different from the one under geometric?

Reply:
Good catch! Think I was trying to make it generic initially, but then realized the metrics are specific to geometric. That file was outdated, deleted it. [0e98f3a](https://github.com/Layr-Labs/eigensdk-go/pull/284/commits/0e98f3a4632fd84ba52c748ac167702561503c39)